modules/installation-configuration-parameters.adoc

```diff
@@ -663,6 +684,47 @@ For production {product-title} clusters on which you want to perform installatio
 ====
 a|For example, `sshKey: ssh-ed25519 AAAA..`.
 
+ifdef::ibm-power-vs[]
+|`platform.powervs.vpcRegion`
+|Specifies the IBM Cloud region in which to create VPC resources.
+|String. For example `existing_vpc_region`.
+
+|`platform.powervs.vpcSubnets`
+|Specifies existing subnets (by name) where cluster resources will be created.
+|String. For example `powervs_region_example_subnet`.
+
+|`platform.powervs.vpcName`
+|Specifies the IBM Cloud VPC name.
+|String. For example `existing_vpcName`.
+
+|`platform.powervs.cloudConnectionName`
+|The CloudConnctionName is the name of an existing PowerVS Cloud connection.
```

Comment:
Not sure, but is "CloudConnctionName" above missing an `e`?

Reply:
yes, corrected. Thanks.